README.md

```diff
@@ -10,56 +10,71 @@ Swiftly generate autolayout constraints.
 
 To run the example project, simply run `pod try swiftly`. Alternatively, you can clone the repo and run the project in the example directory.
 
-All `UIView`s and `UILayoutGuide`s respond to the `applyLayout` method which takes a variadic list of `Swiftly` objects. Convenience initializers are available which pair with all of Apple's `NSLayoutAttribute` types. Common combinatorial layout types `Flush`, `FlushToMargins`, `Vertical`, `Horizontal`, `Center`, and `Size` are also available.
+All `UIView`s and `UILayoutGuide`s respond to the `applyLayout` method which takes a variadic list of `Swiftly` objects. Convenience initializers are available which pair with all of Apple's `NSLayoutAttribute` types. Common combinatorial layout types `flush`, `flushToMargins`, `vertical`, `horizontal`, `center`, and `size` are also available.
 
 ```swift
-view.applyLayout(.CenterX(), .Vertical(), .Width() * 0.5)
+view.applyLayout(.centerX(), .vertical(), .width() * 0.5)
 ```
 
 ## Installation
 
-Swiftly is available through [CocoaPods](http://cocoapods.org). To install
+### Cocoapods
+
+Swiftly is available through [CocoaPods](https://cocoapods.org). To install
 it, simply add the following line to your Podfile:
 
 ```ruby
 pod "Swiftly"
 ```
 
+### Carthage
+
+Swiftly is available through [Carthage](Swiftly/Swiftly.swift). To install
+it, simply add the following line to your Cartfile:
+
+```
```

Comment:
In case GitHub ever adds highlighting for Carthage files, I like adding the language tag here: `ogdl`.